### PR TITLE
chore: migrate articles to new screen

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [] 

### Description

This PR migrates the articles screen to use the new screen component 

**video recorded after: d1e9e48**

https://github.com/artsy/eigen/assets/11945712/9f625640-81f6-475d-bfa1-3f9179f4b9ee

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Use new screen in articles screen - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Review by gkartalis — DISMISSED** [review]

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Use new screen in articles screen - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 97bda6807ebe8c1c59d5a381bb59eb433152c85c
</p>

**Comment by damassi** [comment]
I know this is part of the overall screen layout, but at first glance this title header text seems too close to the image 🤔  

Screenshot 2024-02-28 at 8 23 21 AM


@MounirDhahri could you paste a screen cap of another similar screen? Curious what it looks like in comparison.

**Comment by MounirDhahri** [comment]
You're right, other screens have more margin. I will fix this
Simulator Screenshot - iPhone 15 Pro - 2024-02-28 at 17 29 48

**Comment by damassi** [comment]
Thank you! Once that's in, 👍 👍

File: src/app/Scenes/Articles/Articles.tsx [hunk 1]
```diff
@@ -51,11 +51,13 @@ export const Articles: React.FC = () => {
   )
 }
 
-export const ArticlesScreen: React.FC = () => (
-  <Suspense fallback={<ArticlesPlaceholder />}>
-    <Articles />
-  </Suspense>
-)
+export const ArticlesScreen: React.FC = () => {
+  return (
```
Comment (damassi) [inline src/app/Scenes/Articles/Articles.tsx:55]: 😭 thank you. Its the little things...

**Pull request merged**